templates/OperationResponseBodySnippet.stencil

```diff
@@ -1,24 +1,27 @@
 
 switch result {
 case let .success(data):
-    guard let data = data else {
-        let noDataError = AzureError.sdk("Response data expected but not found.")
-        DispatchQueue.main.async {
-            completionHandler(.failure(noDataError), httpResponse)
-        }
-        return
-    }
-    do {
-        let decoder = JSONDecoder()
-        let decoded = try decoder.decode({{ op.returnType.name }}.self, from: data)
-        DispatchQueue.main.async {
-            completionHandler(.success(decoded), httpResponse)
-        }
-    } catch {
-        DispatchQueue.main.async {
-            completionHandler(.failure(AzureError.sdk("Decoding error.", error)), httpResponse)
+    {% include "OperationResponseData.stencil" %}
+    let statusCode = httpResponse?.statusCode ?? -1
```

Comment:
there is a new snippet in  {% include "OperationResponseData.stencil" %}. which should be refactored into a new snippet?

Reply:
It seems it should be something like:
```Swift
guard let statusCode = httpResponse?.statusCode else {
  completionHandler(.failure("Expected a status code in response but didn't find one.")
}
```

I've seen the status code one-liner in a couple spots, so it might make sense to have a snippet for this.